docs/data/toolpad/core/components/use-notifications/use-notifications.md

```diff
@@ -4,9 +4,9 @@ title: useNotifications
 components: NotificationsProvider
 ---
 
-# Notifications
+# useNotifications
```

Comment:
I think we can keep "Notifications" and instead change the `title` property above